pkgs/top-level/python-packages.nix

```
@@ -2406,7 +2406,7 @@ in {
 
   pkginfo = callPackage ../development/python-modules/pkginfo { };
 
-  pre-commit = callPackage ../development/python-modules/pre-commit { };
+  pre-commit = disabledIf isPy27 (callPackage ../development/python-modules/pre-commit { });
```

I would move disabled logic into the expression
```suggestion
  pre-commit = callPackage ../development/python-modules/pre-commit { };
```